pom.xml

```diff
@@ -51,6 +51,17 @@
             <version>1.5.11</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
```

Comment:
Even though lombok is very helpful and used in large projects for this exercise we would recommend not using it in order to practice coding and best practices. Also when using lombok there are several drawbacks as well so it shouldn't be used as a fits all solution. If you want you can read more about this subject [here ](https://medium.com/@vgonzalo/dont-use-lombok-672418daa81) (even though some drawbacks have be addressed since this article was published)